index.bs

```diff
@@ -10710,6 +10495,40 @@ property-installation style as namespaces.)
     1.  Return |F|.
 </div>
 
+
+<h5 id="es-default-operations">Default operations</h5>
+
+Only [=regular operations=] which have a <dfn>corresponding default operation</dfn> defined below
+may be declared with a [{{Default}}] [=extended attribute=].
+
+
+<h6 id="es-default-tojson">default toJSON operation</h5>
+
+The [=corresponding default operation=] of the “toJSON” operation is the [=default toJSON operation=].
+
+<div algorithm>
+
+    To invoke the <dfn export>default toJSON operation</dfn> of [=interface=] |I|,
+    run the the following steps:
+
+    1.  Let |O| be the <emu-val>this</emu-val> value.
+    1.  Let |result| be [=!=] [=ObjectCreate=]([=%ObjectPrototype%=]).
```

Comment:
Yeah. Let/set algo-writing constraints pushed me to write it that way. An implementation obviously shouldn't create the object if it's not needed.

I guess I'll just either:

```
1.  Let |result| be null;
1.  If, … set |result| to …
1.  Otherwise, set |result| to …
```
or:

```
1.  If, … let |result| to …
1.  Otherwise, let |result| to …
```

Which one would you favor?